lxd/devices.go

```diff
@@ -101,6 +101,65 @@ type cardIds struct {
 	pci string
 }
 
+// Fallback for old drivers which don't provide "Device Minor:"
+func findNvidiaMinorOld() (string, error) {
```

Comment:
Does this presuppose that there is not just one Nvidia card but also one `/dev/dri/card<idx>` card?

Reply:
I don't think so, but I wouldn't know.  I did test a bit on two machines with two entries in /dev/dri/card* (an nvidia card and builtin intel graphics) and it seemed to work fine with the old and new nvidia cards I tried.